.github/dependabot.yml

```diff
@@ -0,0 +1,9 @@
+# Set update schedule for GitHub actions and other package managers
+
+version: 2
+updates:
+  - package-ecosystem: "github-actions"
+    directory: "/"
+    schedule:
+      # Check for updates to GitHub Actions every week
+      interval: "weekly"
```

Comment:
@geoffreyssh97 
Is there a reason we have the
```
  - package-ecosystem: "swift"
    directory: "/"
    schedule:
      interval: "weekly"
    open-pull-requests-limit: 5
```
step on some of these repos and not others?